src/gui/guiFormSpecMenu.cpp

```diff
@@ -1776,6 +1964,11 @@ void GUIFormSpecMenu::parseAnchor(parserData *data, const std::string &element)
 			<< "'" << std::endl;
 }
 
+void GUIFormSpecMenu::parseRealCoordinates(parserData* data, const std::string
+	&element) {
+	data->real_coordinates = is_yes(element);
```

Comment:
Is the parameter needed? Even if `false` would be used in mods to ensure compatibility in the future; old mods would not make use of this element anyway.

Reply:
There might be possible use cases for the old system, and it doesn't hurt even if it isn't ever used.  I'll wait for more input before changing this.

Inlining sounds good, though.  The function was from before I knew about `is_yes`.